templates/taxi/driver_list.html

```diff
@@ -1,17 +1,22 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Driver list</h1>
+  <h1 class="taxi-title">Drivers</h1>
   {% if driver_list %}
-    <ul>
-      {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+    <ul class="driver-list">
+      {% for driver in driver_list %}
+        <li class="driver-item">
+          <a href="{{ driver.get_absolute_url }}">
```

Comment:
Ensure that the `Driver` model has a `get_absolute_url` method implemented. This method should return the URL to the driver's detail page to avoid broken links.